scss/blocks/figma-component-props-table.scss

```diff
@@ -1,8 +1,9 @@
 :root {
+    --figma-component-border-color: #ECEEF3;
+    --figma-component-text-color: #14171F;
+    --figma-component-light-grey-color: #606981;
+
     --figma-component-props-table-background-color: rgba(244, 245, 247, 0.95);
-    --figma-component-props-table-border-color: #ECEEF3;
-    --figma-component-props-table-light-grey-color: #606981;
-    --figma-component-props-table-text-color: #14171F;
     --figma-component-props-table-component-name-color: #000000;
     --figma-component-props-table-fixed-header-width: 224px;
```

Comment:
The raw values should not really happen anywhere below the core theme if even slightly possibly + the convention of all other CSS variables in docs is using camelCase.

```
--figmaComponentBorderColor: var(--colorBorderSubtle);
--figmaComponentInk: var(--colorInkPrimary);
--figmaComponentInkSecondary: var(--colorInkSecondary);
--figmaComponentPropsTableHeaderBackround :var(--colorBackgroundSecondary) (original --figma-component-props-table-background-color)
```

`--figma-component-props-table-component-name-color` →  not needed, migrate to `--figmaComponentInk`

Reply:
@mbednarik This doesn't have response yet, and I see it's wrong also in the other PR